tools/scalajsenv.js

```diff
@@ -272,7 +264,7 @@ ScalaJS.objectGetClass = function(instance) {
       return ScalaJS.d.sr_BoxedUnit.getClassOf();
     default:
       if (instance === null)
-        ScalaJS.throwNullPointerException();
+        return instance.getClass__jl_Class();
```

Comment:
`instance.getClass__jl_Class()` will necessarily throw because `instance` is `null` at this point. The `return` is therefore dead code, strictly speaking, but it is there by symmetry with the other branches. This is a pattern that is often used in this file. See e.g. in `objectClone` just below.